app/controllers/api/beta/search_controller.rb

```diff
@@ -7,7 +7,6 @@ class SearchController < ApiController
         :heading_statistics,
         :chapter_statistics,
         :guide,
-        'facet_filter_statistics',
```

Comment:
This actually isn't required/is implicit when we added `'facet_filter_statistics.facet_classification_statistics'` as an include